corpus/data_augmentation_audiomentations.py

```diff
@@ -0,0 +1,127 @@
+__all__ = ["AddImpulseResponseJob", "MixNoiseJob"]
+
+import os
+import audiomentations as am
+import shutil
+import logging
+from scipy.io import wavfile
+
+
+from sisyphus import *
+from recipe.i6_core.lib import corpus
+
+class AddImpulseResponseJob(Job):
+    def __init__(self, corpus_file, new_corpus_name, ir_path, p):
+        self.corpus_file = corpus_file
+        self.new_corpus_name = new_corpus_name
+        self.impulse_responses_dir = ir_path 
+        self.p_impulse_response = p
+
+        self.out_audio_folder = self.output_path("out_audio/", directory=True)
+        self.out_corpus = self.output_path("ir_convoluted.corpus.xml.gz")
+        self.out_segment_file = self.output_path("ir_convoluted.segments")
```

Comment:
This segment file does not provide any additional information. In the spirit of "one job does exactly one thing" I would vote to remove it here and instead obtain it from a `SegmentCorpusJob`.